packages/object-prototype/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "object-prototype",
+  "version": "3.0.2",
+  "description": "A replacement prototype for `Object.prototype` with all the same functions",
+  "main": "index.js",
+  "dependencies": {
+    "object-prototype-functions": "1.2.0"
+  },
+  "devDependencies": {
+    "debug": "^4.1.1",
+    "tape": "^4.13.0"
+  },
+  "scripts": {
+    "test": "node test.js"
```

Comment:
@kobelb FYI: These tests can be run from within the `packages/object-prototype` directory manually but are not run as part of the regular jest/mocha tests (as they don't use neither jest nor mocha).

Reply:
Ah, I see that the tests are actually run contrary to what's stated in the `packages/README.md` file. Great! In that case, I do need to fix them to run in slobby mode.